mandos/src/esdt.rs

```diff
@@ -0,0 +1,42 @@
+use super::*;
+use std::collections::BTreeMap;
+
+#[derive(Debug)]
+pub struct Esdt {
+    pub token_identifier: Option<BytesValue>,
```

Comment:
Should this be an Option?

Reply:
I think Alin committed this here by accident. It's ok, we'll return on this.